locations/spiders/bealls_florida.py

```diff
@@ -0,0 +1,46 @@
+from re import search
+from json import loads
+from urllib.parse import unquote
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class BeallsFlorida(SitemapSpider, StructuredDataSpider):
+    name = "bealls_florida"
```

Comment:
If this is a US brand only, the class name should be `BeallsFloridaUSSpider` and `name = "bealls_florida_us"`

Reply:
I looked to other spiders and see the name also includes the country code, I'll rename this one as well. thanks!